docs/SNAPD_CONFIGURATION.md

```diff
@@ -216,6 +216,15 @@ The same configuration settings above can also be provided in a JSON formatted c
 }
 ```
 
+## Restarting snapd to pick up configuration changes
+If changes are made to the configuration file, `snapd` must be restarted to pick up those changes. Fortunately, this is a simple matter of sending a `SIGHUP` signal to the `snapd` process. For example, the following command will restart the `snapd` process on the local system:
+
+```bash
+$ kill -HUP `pidof snapd`
+```
+
+Note that in this example, we are using the `pidof` command to retrieve the process ID of the `snapd` process. If the `pidof` command is not available on your system you might have to use a `ps aux` command and pipe the output of that command to a `grep snapd` command in order to obtain the process ID of the `snapd` process. Once the `snapd` process receives that signal it will restart and pick up any changes that have been made to the configuration file that was originally used to Âstart the `snapd` process.
+
```

Comment:
@tjmcs I know it's implied but do you think we should mention that plugins and tasks that were loaded/created/started after snapd started will not be preserved?